vms/rpcchainvm/factory.go

```diff
@@ -49,15 +49,16 @@ func (f *factory) New(log logging.Logger) (interface{}, error) {
 		Log:              log,
 	}
 
-	listener, err := grpcutils.NewListener()
+	// TODO update interface to accept context
+	listener, err := grpcutils.NewListener(context.TODO())
 	if err != nil {
 		return nil, fmt.Errorf("failed to create listener: %w", err)
 	}
 
 	status, stopper, err := subprocess.Bootstrap(
 		context.TODO(),
 		listener,
-		subprocess.NewCmd(f.path),
+		subprocess.NewCmd(context.TODO(), f.path),
```

Comment:
I'm not sure that `context.TODO()` is correct here. If we were to take a context in to `New`, we _must not_ provide that context to this command, as then the caller's context getting cancelled would kill the sub-process, even after this function has returned successfully.

The prior behavior where this was `context.Background` seems to be more correct.